builtin/providers/pagerduty/config.go

```diff
@@ -11,10 +12,31 @@ type Config struct {
 	Token string
 }
 
+const invalidCredentials = `
+
+No valid credentials found for PagerDuty provider.
+Please see https://www.terraform.io/docs/providers/pagerduty/index.html
+for more information on providing credentials for this provider.
+`
+
 // Client returns a new PagerDuty client
 func (c *Config) Client() (*pagerduty.Client, error) {
+	// Validate that the PagerDuty token is set
+	if c.Token == "" {
+		return nil, fmt.Errorf(invalidCredentials)
+	}
+
 	client := pagerduty.NewClient(c.Token)
 
+	// Validate the credentials by calling the abilities endpoint,
+	// if we get a 401 response back we return an error to the user
+	if _, err := client.ListAbilities(); err != nil {
```

Comment:
Added a `ignore_credentials_validation` field (similar to the one in the AWS provider) so the user can skip the validation against the PagerDuty API :)